.github/workflows/Build.yml

```diff
@@ -91,6 +91,10 @@ jobs:
       run: docker exec kphp-build-container-${{matrix.os}} bash -c
               "composer install -d ${{env.kphp_polyfills_dir}}"
 
+    - name: Install python packages
```

Comment:
Why not do this while building a Docker image?

Reply:
The main reason is that we have 3 targets with same package versions, so we have installation in one file, which is also good for debugging Github Actions and for adding new targets.

Reply:
During the local debug, we use these Dockerfiles, we don't want to do unnecessary things to run tests...

Reply:
Done.